js/src/util/index.js

```diff
@@ -135,33 +135,15 @@ const typeCheckConfig = (componentName, config, configTypes) => {
 }
 
 const isVisible = element => {
-  const eligibleElement = el => el && el.style
-
-  if (!eligibleElement(element)) {
-    return false
-  }
-
-  if (element.tagName !== 'AREA' && element.getClientRects().length === 0) {
+  if (
+    !element ||
```

Comment:
```suggestion

```

I think `isElement()` check is enough

Reply:
Unfortunately `isElement()` expects a non-null parameter, and there is an existing `isVisible()` test requiring safe handling of a `null|undefined` parameter. While adding support for a null parameter to `isElement()` seems like the right thing to do, I'd imagine it should be done outside this PR for clarity purposes.

Reply:
Are you sure? :thinking: 

https://github.com/twbs/bootstrap/blob/de0b797ab2bb80e4576c15f1992dc5755378edcd/js/src/util/index.js#L106
Looks to work


![image](https://user-images.githubusercontent.com/22406063/118936794-8898b500-b955-11eb-978a-103d67e0713b.png)

Reply:
oh lol - this branch has not been synced since you fixed it in #33327, thanks! I'll sync and update 👍